[2023-07-17 12:11:06,051] {processor.py:153} INFO - Started process (PID=10380) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:11:06,052] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:11:06,053] {logging_mixin.py:115} INFO - [2023-07-17 12:11:06,053] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:11:06,090] {logging_mixin.py:115} INFO - [2023-07-17 12:11:06,089] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:11:06,090] {logging_mixin.py:115} INFO - [2023-07-17 12:11:06,090] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:11:06,090] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:11:06,113] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2023-07-17 12:11:43,267] {processor.py:153} INFO - Started process (PID=10735) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:11:43,271] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:11:43,273] {logging_mixin.py:115} INFO - [2023-07-17 12:11:43,273] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:11:43,319] {logging_mixin.py:115} INFO - [2023-07-17 12:11:43,318] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:11:43,320] {logging_mixin.py:115} INFO - [2023-07-17 12:11:43,319] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:11:43,320] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:11:43,353] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.093 seconds
[2023-07-17 12:12:22,722] {processor.py:153} INFO - Started process (PID=10818) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:12:22,726] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:12:22,728] {logging_mixin.py:115} INFO - [2023-07-17 12:12:22,728] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:12:22,785] {logging_mixin.py:115} INFO - [2023-07-17 12:12:22,784] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:12:22,786] {logging_mixin.py:115} INFO - [2023-07-17 12:12:22,785] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:12:22,786] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:12:22,824] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.113 seconds
[2023-07-17 12:13:02,753] {processor.py:153} INFO - Started process (PID=10998) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:13:02,755] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:13:02,757] {logging_mixin.py:115} INFO - [2023-07-17 12:13:02,756] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:13:02,795] {logging_mixin.py:115} INFO - [2023-07-17 12:13:02,795] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:13:02,796] {logging_mixin.py:115} INFO - [2023-07-17 12:13:02,796] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:13:02,796] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:13:02,823] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.076 seconds
[2023-07-17 12:13:45,077] {processor.py:153} INFO - Started process (PID=11117) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:13:45,081] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:13:45,082] {logging_mixin.py:115} INFO - [2023-07-17 12:13:45,081] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:13:45,123] {logging_mixin.py:115} INFO - [2023-07-17 12:13:45,122] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:13:45,124] {logging_mixin.py:115} INFO - [2023-07-17 12:13:45,123] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:13:45,124] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:13:45,148] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.078 seconds
[2023-07-17 12:14:24,536] {processor.py:153} INFO - Started process (PID=11156) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:14:24,540] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:14:24,541] {logging_mixin.py:115} INFO - [2023-07-17 12:14:24,541] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:14:24,582] {logging_mixin.py:115} INFO - [2023-07-17 12:14:24,580] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:14:24,582] {logging_mixin.py:115} INFO - [2023-07-17 12:14:24,582] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:14:24,582] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:14:24,608] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.079 seconds
[2023-07-17 12:15:04,562] {processor.py:153} INFO - Started process (PID=11269) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:15:04,566] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:15:04,567] {logging_mixin.py:115} INFO - [2023-07-17 12:15:04,567] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:15:04,608] {logging_mixin.py:115} INFO - [2023-07-17 12:15:04,607] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:15:04,608] {logging_mixin.py:115} INFO - [2023-07-17 12:15:04,608] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:15:04,608] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:15:04,635] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.080 seconds
[2023-07-17 12:15:45,233] {processor.py:153} INFO - Started process (PID=11690) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:15:45,237] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:15:45,238] {logging_mixin.py:115} INFO - [2023-07-17 12:15:45,238] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:15:45,282] {logging_mixin.py:115} INFO - [2023-07-17 12:15:45,281] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:15:45,283] {logging_mixin.py:115} INFO - [2023-07-17 12:15:45,283] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:15:45,283] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:15:45,307] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.083 seconds
[2023-07-17 12:16:24,922] {processor.py:153} INFO - Started process (PID=11748) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:16:24,926] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:16:24,927] {logging_mixin.py:115} INFO - [2023-07-17 12:16:24,927] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:16:24,971] {logging_mixin.py:115} INFO - [2023-07-17 12:16:24,970] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:16:24,971] {logging_mixin.py:115} INFO - [2023-07-17 12:16:24,971] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:16:24,972] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:16:24,999] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.083 seconds
[2023-07-17 12:17:06,165] {processor.py:153} INFO - Started process (PID=11821) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:17:06,169] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:17:06,170] {logging_mixin.py:115} INFO - [2023-07-17 12:17:06,169] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:17:06,211] {logging_mixin.py:115} INFO - [2023-07-17 12:17:06,210] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:17:06,212] {logging_mixin.py:115} INFO - [2023-07-17 12:17:06,212] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:17:06,212] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:17:06,243] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.085 seconds
[2023-07-17 12:17:45,045] {processor.py:153} INFO - Started process (PID=11921) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:17:45,046] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:17:45,047] {logging_mixin.py:115} INFO - [2023-07-17 12:17:45,047] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:17:45,091] {logging_mixin.py:115} INFO - [2023-07-17 12:17:45,090] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:17:45,092] {logging_mixin.py:115} INFO - [2023-07-17 12:17:45,092] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:17:45,092] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:17:45,117] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.081 seconds
[2023-07-17 12:18:23,647] {processor.py:153} INFO - Started process (PID=11990) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:18:23,648] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:18:23,650] {logging_mixin.py:115} INFO - [2023-07-17 12:18:23,649] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:18:23,693] {logging_mixin.py:115} INFO - [2023-07-17 12:18:23,692] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:18:23,693] {logging_mixin.py:115} INFO - [2023-07-17 12:18:23,693] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:18:23,694] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:18:23,724] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.084 seconds
[2023-07-17 12:19:07,449] {processor.py:153} INFO - Started process (PID=12057) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:19:07,453] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:19:07,454] {logging_mixin.py:115} INFO - [2023-07-17 12:19:07,453] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:19:07,495] {logging_mixin.py:115} INFO - [2023-07-17 12:19:07,494] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:19:07,496] {logging_mixin.py:115} INFO - [2023-07-17 12:19:07,495] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:19:07,496] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:19:07,523] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.081 seconds
[2023-07-17 12:19:49,369] {processor.py:153} INFO - Started process (PID=12132) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:19:49,373] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:19:49,374] {logging_mixin.py:115} INFO - [2023-07-17 12:19:49,374] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:19:49,415] {logging_mixin.py:115} INFO - [2023-07-17 12:19:49,414] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:19:49,415] {logging_mixin.py:115} INFO - [2023-07-17 12:19:49,415] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:19:49,415] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:19:49,440] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.077 seconds
[2023-07-17 12:20:33,039] {processor.py:153} INFO - Started process (PID=12191) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:20:33,043] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:20:33,044] {logging_mixin.py:115} INFO - [2023-07-17 12:20:33,044] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:20:33,080] {logging_mixin.py:115} INFO - [2023-07-17 12:20:33,079] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:20:33,080] {logging_mixin.py:115} INFO - [2023-07-17 12:20:33,080] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:20:33,080] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:20:33,103] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.070 seconds
[2023-07-17 12:21:11,372] {processor.py:153} INFO - Started process (PID=12301) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:21:11,373] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:21:11,374] {logging_mixin.py:115} INFO - [2023-07-17 12:21:11,374] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:21:11,409] {logging_mixin.py:115} INFO - [2023-07-17 12:21:11,408] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:21:11,409] {logging_mixin.py:115} INFO - [2023-07-17 12:21:11,409] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:21:11,409] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:21:11,429] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.063 seconds
[2023-07-17 12:21:46,759] {processor.py:153} INFO - Started process (PID=12361) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:21:46,760] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:21:46,762] {logging_mixin.py:115} INFO - [2023-07-17 12:21:46,761] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:21:46,806] {logging_mixin.py:115} INFO - [2023-07-17 12:21:46,805] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:21:46,806] {logging_mixin.py:115} INFO - [2023-07-17 12:21:46,806] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:21:46,807] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:21:46,831] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.078 seconds
[2023-07-17 12:22:23,521] {processor.py:153} INFO - Started process (PID=12717) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:22:23,524] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:22:23,525] {logging_mixin.py:115} INFO - [2023-07-17 12:22:23,525] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:22:23,540] {logging_mixin.py:115} INFO - [2023-07-17 12:22:23,540] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:22:23,540] {logging_mixin.py:115} INFO - [2023-07-17 12:22:23,540] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:22:23,540] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:22:23,551] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-07-17 12:23:01,198] {processor.py:153} INFO - Started process (PID=12767) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:23:01,202] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:23:01,203] {logging_mixin.py:115} INFO - [2023-07-17 12:23:01,203] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:23:01,238] {logging_mixin.py:115} INFO - [2023-07-17 12:23:01,237] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:23:01,238] {logging_mixin.py:115} INFO - [2023-07-17 12:23:01,238] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:23:01,238] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:23:01,259] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2023-07-17 12:23:42,811] {processor.py:153} INFO - Started process (PID=12820) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:23:42,812] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:23:42,813] {logging_mixin.py:115} INFO - [2023-07-17 12:23:42,813] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:23:42,848] {logging_mixin.py:115} INFO - [2023-07-17 12:23:42,847] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:23:42,849] {logging_mixin.py:115} INFO - [2023-07-17 12:23:42,848] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:23:42,849] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:23:42,869] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.063 seconds
[2023-07-17 12:24:19,507] {processor.py:153} INFO - Started process (PID=12875) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:24:19,509] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:24:19,510] {logging_mixin.py:115} INFO - [2023-07-17 12:24:19,509] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:24:19,553] {logging_mixin.py:115} INFO - [2023-07-17 12:24:19,552] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:24:19,553] {logging_mixin.py:115} INFO - [2023-07-17 12:24:19,553] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:24:19,554] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:24:19,575] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.075 seconds
[2023-07-17 12:24:56,557] {processor.py:153} INFO - Started process (PID=12930) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:24:56,558] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:24:56,559] {logging_mixin.py:115} INFO - [2023-07-17 12:24:56,558] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:24:56,593] {logging_mixin.py:115} INFO - [2023-07-17 12:24:56,592] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:24:56,593] {logging_mixin.py:115} INFO - [2023-07-17 12:24:56,593] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:24:56,593] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:24:56,613] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.062 seconds
[2023-07-17 12:25:37,971] {processor.py:153} INFO - Started process (PID=13042) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:25:37,972] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:25:37,973] {logging_mixin.py:115} INFO - [2023-07-17 12:25:37,973] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:25:38,007] {logging_mixin.py:115} INFO - [2023-07-17 12:25:38,006] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:25:38,007] {logging_mixin.py:115} INFO - [2023-07-17 12:25:38,007] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:25:38,007] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:25:38,026] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.061 seconds
[2023-07-17 12:26:12,199] {processor.py:153} INFO - Started process (PID=13182) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:26:12,202] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:26:12,203] {logging_mixin.py:115} INFO - [2023-07-17 12:26:12,203] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:26:12,236] {logging_mixin.py:115} INFO - [2023-07-17 12:26:12,235] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:26:12,237] {logging_mixin.py:115} INFO - [2023-07-17 12:26:12,237] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:26:12,237] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:26:12,257] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.064 seconds
[2023-07-17 12:26:48,652] {processor.py:153} INFO - Started process (PID=13237) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:26:48,656] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:26:48,657] {logging_mixin.py:115} INFO - [2023-07-17 12:26:48,657] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:26:48,691] {logging_mixin.py:115} INFO - [2023-07-17 12:26:48,691] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:26:48,692] {logging_mixin.py:115} INFO - [2023-07-17 12:26:48,692] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:26:48,692] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:26:48,713] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2023-07-17 12:27:25,629] {processor.py:153} INFO - Started process (PID=13303) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:27:25,633] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:27:25,634] {logging_mixin.py:115} INFO - [2023-07-17 12:27:25,633] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:27:25,668] {logging_mixin.py:115} INFO - [2023-07-17 12:27:25,667] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:27:25,669] {logging_mixin.py:115} INFO - [2023-07-17 12:27:25,669] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:27:25,669] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:27:25,691] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2023-07-17 12:28:04,576] {processor.py:153} INFO - Started process (PID=13363) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:28:04,577] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:28:04,578] {logging_mixin.py:115} INFO - [2023-07-17 12:28:04,578] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:28:04,613] {logging_mixin.py:115} INFO - [2023-07-17 12:28:04,612] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:28:04,613] {logging_mixin.py:115} INFO - [2023-07-17 12:28:04,613] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:28:04,614] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:28:04,635] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2023-07-17 12:28:44,831] {processor.py:153} INFO - Started process (PID=13423) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:28:44,835] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:28:44,836] {logging_mixin.py:115} INFO - [2023-07-17 12:28:44,835] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:28:44,870] {logging_mixin.py:115} INFO - [2023-07-17 12:28:44,869] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:28:44,870] {logging_mixin.py:115} INFO - [2023-07-17 12:28:44,870] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:28:44,871] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:28:44,892] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2023-07-17 12:29:20,460] {processor.py:153} INFO - Started process (PID=13466) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:29:20,463] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:29:20,464] {logging_mixin.py:115} INFO - [2023-07-17 12:29:20,464] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:29:20,497] {logging_mixin.py:115} INFO - [2023-07-17 12:29:20,497] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:29:20,498] {logging_mixin.py:115} INFO - [2023-07-17 12:29:20,498] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:29:20,498] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:29:20,519] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2023-07-17 12:29:57,698] {processor.py:153} INFO - Started process (PID=13674) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:29:57,702] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:29:57,703] {logging_mixin.py:115} INFO - [2023-07-17 12:29:57,702] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:29:57,736] {logging_mixin.py:115} INFO - [2023-07-17 12:29:57,736] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:29:57,737] {logging_mixin.py:115} INFO - [2023-07-17 12:29:57,737] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:29:57,737] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:29:57,758] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2023-07-17 12:30:33,229] {processor.py:153} INFO - Started process (PID=13722) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:30:33,233] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:30:33,234] {logging_mixin.py:115} INFO - [2023-07-17 12:30:33,233] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:30:33,261] {logging_mixin.py:115} INFO - [2023-07-17 12:30:33,260] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:30:33,261] {logging_mixin.py:115} INFO - [2023-07-17 12:30:33,261] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:30:33,261] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:30:33,269] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2023-07-17 12:31:13,956] {processor.py:153} INFO - Started process (PID=13862) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:31:13,960] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:31:13,961] {logging_mixin.py:115} INFO - [2023-07-17 12:31:13,961] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:31:13,995] {logging_mixin.py:115} INFO - [2023-07-17 12:31:13,994] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:31:13,995] {logging_mixin.py:115} INFO - [2023-07-17 12:31:13,995] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:31:13,995] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:31:14,017] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2023-07-17 12:31:50,193] {processor.py:153} INFO - Started process (PID=14050) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:31:50,197] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:31:50,198] {logging_mixin.py:115} INFO - [2023-07-17 12:31:50,198] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:31:50,232] {logging_mixin.py:115} INFO - [2023-07-17 12:31:50,231] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:31:50,232] {logging_mixin.py:115} INFO - [2023-07-17 12:31:50,232] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:31:50,233] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:31:50,253] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2023-07-17 12:32:27,104] {processor.py:153} INFO - Started process (PID=14198) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:32:27,105] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:32:27,106] {logging_mixin.py:115} INFO - [2023-07-17 12:32:27,106] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:32:27,145] {logging_mixin.py:115} INFO - [2023-07-17 12:32:27,144] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:32:27,145] {logging_mixin.py:115} INFO - [2023-07-17 12:32:27,145] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:32:27,146] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:32:27,167] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2023-07-17 12:33:05,003] {processor.py:153} INFO - Started process (PID=14257) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:33:05,007] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:33:05,008] {logging_mixin.py:115} INFO - [2023-07-17 12:33:05,007] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:33:05,043] {logging_mixin.py:115} INFO - [2023-07-17 12:33:05,042] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:33:05,044] {logging_mixin.py:115} INFO - [2023-07-17 12:33:05,044] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:33:05,044] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:33:05,066] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2023-07-17 12:33:47,555] {processor.py:153} INFO - Started process (PID=14313) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:33:47,559] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:33:47,560] {logging_mixin.py:115} INFO - [2023-07-17 12:33:47,560] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:33:47,606] {logging_mixin.py:115} INFO - [2023-07-17 12:33:47,605] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:33:47,606] {logging_mixin.py:115} INFO - [2023-07-17 12:33:47,606] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:33:47,606] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:33:47,628] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.079 seconds
[2023-07-17 12:34:23,167] {processor.py:153} INFO - Started process (PID=14404) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:34:23,169] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:34:23,170] {logging_mixin.py:115} INFO - [2023-07-17 12:34:23,170] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:34:23,208] {logging_mixin.py:115} INFO - [2023-07-17 12:34:23,207] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:34:23,208] {logging_mixin.py:115} INFO - [2023-07-17 12:34:23,208] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:34:23,208] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:34:23,230] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.070 seconds
[2023-07-17 12:35:04,713] {processor.py:153} INFO - Started process (PID=14490) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:35:04,714] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:35:04,715] {logging_mixin.py:115} INFO - [2023-07-17 12:35:04,715] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:35:04,748] {logging_mixin.py:115} INFO - [2023-07-17 12:35:04,747] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:35:04,748] {logging_mixin.py:115} INFO - [2023-07-17 12:35:04,748] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:35:04,748] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:35:04,769] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.062 seconds
[2023-07-17 12:35:45,411] {processor.py:153} INFO - Started process (PID=14651) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:35:45,412] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:35:45,413] {logging_mixin.py:115} INFO - [2023-07-17 12:35:45,413] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:35:45,451] {logging_mixin.py:115} INFO - [2023-07-17 12:35:45,450] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:35:45,451] {logging_mixin.py:115} INFO - [2023-07-17 12:35:45,451] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:35:45,451] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:35:45,473] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2023-07-17 12:36:25,992] {processor.py:153} INFO - Started process (PID=14727) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:36:25,993] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:36:25,994] {logging_mixin.py:115} INFO - [2023-07-17 12:36:25,994] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:36:26,028] {logging_mixin.py:115} INFO - [2023-07-17 12:36:26,027] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:36:26,028] {logging_mixin.py:115} INFO - [2023-07-17 12:36:26,028] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:36:26,028] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:36:26,050] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.064 seconds
[2023-07-17 12:37:02,716] {processor.py:153} INFO - Started process (PID=14819) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:37:02,720] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:37:02,721] {logging_mixin.py:115} INFO - [2023-07-17 12:37:02,721] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:37:02,759] {logging_mixin.py:115} INFO - [2023-07-17 12:37:02,758] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:37:02,759] {logging_mixin.py:115} INFO - [2023-07-17 12:37:02,759] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:37:02,760] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:37:02,784] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.075 seconds
[2023-07-17 12:37:40,079] {processor.py:153} INFO - Started process (PID=14876) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:37:40,082] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:37:40,083] {logging_mixin.py:115} INFO - [2023-07-17 12:37:40,082] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:37:40,097] {logging_mixin.py:115} INFO - [2023-07-17 12:37:40,097] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:37:40,097] {logging_mixin.py:115} INFO - [2023-07-17 12:37:40,097] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:37:40,097] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:37:40,106] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-07-17 12:38:12,266] {processor.py:153} INFO - Started process (PID=14946) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:38:12,269] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:38:12,269] {logging_mixin.py:115} INFO - [2023-07-17 12:38:12,269] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:38:12,282] {logging_mixin.py:115} INFO - [2023-07-17 12:38:12,282] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:38:12,283] {logging_mixin.py:115} INFO - [2023-07-17 12:38:12,282] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:38:12,283] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:38:12,291] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-07-17 12:38:48,653] {processor.py:153} INFO - Started process (PID=14986) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:38:48,657] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 12:38:48,658] {logging_mixin.py:115} INFO - [2023-07-17 12:38:48,658] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:38:48,692] {logging_mixin.py:115} INFO - [2023-07-17 12:38:48,692] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 12:38:48,693] {logging_mixin.py:115} INFO - [2023-07-17 12:38:48,693] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 12:38:48,693] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 12:38:48,714] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2023-07-17 14:51:41,349] {processor.py:153} INFO - Started process (PID=24098) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:51:41,350] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 14:51:41,351] {logging_mixin.py:115} INFO - [2023-07-17 14:51:41,351] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:51:41,400] {logging_mixin.py:115} INFO - [2023-07-17 14:51:41,399] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 14:51:41,400] {logging_mixin.py:115} INFO - [2023-07-17 14:51:41,400] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 14:51:41,400] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:51:41,424] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.081 seconds
[2023-07-17 14:52:22,553] {processor.py:153} INFO - Started process (PID=24290) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:52:22,557] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 14:52:22,558] {logging_mixin.py:115} INFO - [2023-07-17 14:52:22,558] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:52:22,592] {logging_mixin.py:115} INFO - [2023-07-17 14:52:22,592] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 14:52:22,593] {logging_mixin.py:115} INFO - [2023-07-17 14:52:22,593] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 14:52:22,593] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:52:22,615] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2023-07-17 14:53:03,186] {processor.py:153} INFO - Started process (PID=24460) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:53:03,187] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 14:53:03,188] {logging_mixin.py:115} INFO - [2023-07-17 14:53:03,187] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:53:03,224] {logging_mixin.py:115} INFO - [2023-07-17 14:53:03,223] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 14:53:03,225] {logging_mixin.py:115} INFO - [2023-07-17 14:53:03,225] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 14:53:03,225] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:53:03,246] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2023-07-17 14:53:41,171] {processor.py:153} INFO - Started process (PID=24512) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:53:41,175] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 14:53:41,176] {logging_mixin.py:115} INFO - [2023-07-17 14:53:41,175] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:53:41,210] {logging_mixin.py:115} INFO - [2023-07-17 14:53:41,209] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 14:53:41,210] {logging_mixin.py:115} INFO - [2023-07-17 14:53:41,210] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 14:53:41,210] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:53:41,230] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2023-07-17 14:54:41,794] {processor.py:153} INFO - Started process (PID=25080) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:54:41,798] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 14:54:41,800] {logging_mixin.py:115} INFO - [2023-07-17 14:54:41,799] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:54:41,835] {logging_mixin.py:115} INFO - [2023-07-17 14:54:41,834] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 14:54:41,835] {logging_mixin.py:115} INFO - [2023-07-17 14:54:41,835] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 14:54:41,836] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:54:41,858] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2023-07-17 14:55:43,687] {processor.py:153} INFO - Started process (PID=25533) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:55:43,691] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 14:55:43,693] {logging_mixin.py:115} INFO - [2023-07-17 14:55:43,693] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:55:43,732] {logging_mixin.py:115} INFO - [2023-07-17 14:55:43,731] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 14:55:43,732] {logging_mixin.py:115} INFO - [2023-07-17 14:55:43,732] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 14:55:43,732] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:55:43,754] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.073 seconds
[2023-07-17 14:56:23,131] {processor.py:153} INFO - Started process (PID=25638) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:56:23,135] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 14:56:23,136] {logging_mixin.py:115} INFO - [2023-07-17 14:56:23,136] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:56:23,183] {logging_mixin.py:115} INFO - [2023-07-17 14:56:23,182] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 14:56:23,183] {logging_mixin.py:115} INFO - [2023-07-17 14:56:23,183] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 14:56:23,184] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:56:23,211] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.086 seconds
[2023-07-17 14:57:07,613] {processor.py:153} INFO - Started process (PID=26865) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:57:07,613] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 14:57:07,614] {logging_mixin.py:115} INFO - [2023-07-17 14:57:07,614] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:57:07,627] {logging_mixin.py:115} INFO - [2023-07-17 14:57:07,627] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 14:57:07,627] {logging_mixin.py:115} INFO - [2023-07-17 14:57:07,627] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 14:57:07,627] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:57:07,635] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-07-17 14:57:50,551] {processor.py:153} INFO - Started process (PID=27720) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:57:50,555] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 14:57:50,556] {logging_mixin.py:115} INFO - [2023-07-17 14:57:50,556] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:57:50,589] {logging_mixin.py:115} INFO - [2023-07-17 14:57:50,589] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 14:57:50,590] {logging_mixin.py:115} INFO - [2023-07-17 14:57:50,590] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 14:57:50,590] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:57:50,610] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2023-07-17 14:58:34,241] {processor.py:153} INFO - Started process (PID=27787) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:58:34,245] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 14:58:34,246] {logging_mixin.py:115} INFO - [2023-07-17 14:58:34,246] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:58:34,277] {logging_mixin.py:115} INFO - [2023-07-17 14:58:34,276] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 14:58:34,277] {logging_mixin.py:115} INFO - [2023-07-17 14:58:34,277] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 14:58:34,277] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:58:34,284] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2023-07-17 14:59:14,726] {processor.py:153} INFO - Started process (PID=27914) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:59:14,730] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 14:59:14,731] {logging_mixin.py:115} INFO - [2023-07-17 14:59:14,731] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:59:14,779] {logging_mixin.py:115} INFO - [2023-07-17 14:59:14,778] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 14:59:14,780] {logging_mixin.py:115} INFO - [2023-07-17 14:59:14,780] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 14:59:14,780] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:59:14,802] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.085 seconds
[2023-07-17 14:59:59,827] {processor.py:153} INFO - Started process (PID=28025) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:59:59,831] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 14:59:59,831] {logging_mixin.py:115} INFO - [2023-07-17 14:59:59,831] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:59:59,846] {logging_mixin.py:115} INFO - [2023-07-17 14:59:59,846] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 14:59:59,846] {logging_mixin.py:115} INFO - [2023-07-17 14:59:59,846] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 14:59:59,846] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 14:59:59,854] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-07-17 15:00:38,965] {processor.py:153} INFO - Started process (PID=28085) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 15:00:38,969] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 15:00:38,970] {logging_mixin.py:115} INFO - [2023-07-17 15:00:38,970] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 15:00:39,007] {logging_mixin.py:115} INFO - [2023-07-17 15:00:39,006] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 15:00:39,007] {logging_mixin.py:115} INFO - [2023-07-17 15:00:39,007] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 15:00:39,008] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 15:00:39,037] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.078 seconds
[2023-07-17 15:01:18,056] {processor.py:153} INFO - Started process (PID=28229) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 15:01:18,057] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 15:01:18,058] {logging_mixin.py:115} INFO - [2023-07-17 15:01:18,058] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 15:01:18,094] {logging_mixin.py:115} INFO - [2023-07-17 15:01:18,093] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 15:01:18,094] {logging_mixin.py:115} INFO - [2023-07-17 15:01:18,094] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 15:01:18,094] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 15:01:18,114] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.063 seconds
[2023-07-17 15:02:00,714] {processor.py:153} INFO - Started process (PID=28306) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 15:02:00,718] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 15:02:00,719] {logging_mixin.py:115} INFO - [2023-07-17 15:02:00,719] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 15:02:00,756] {logging_mixin.py:115} INFO - [2023-07-17 15:02:00,756] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 15:02:00,757] {logging_mixin.py:115} INFO - [2023-07-17 15:02:00,757] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 15:02:00,757] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 15:02:00,779] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.072 seconds
[2023-07-17 15:02:44,498] {processor.py:153} INFO - Started process (PID=28371) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 15:02:44,500] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 15:02:44,501] {logging_mixin.py:115} INFO - [2023-07-17 15:02:44,501] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 15:02:44,539] {logging_mixin.py:115} INFO - [2023-07-17 15:02:44,538] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 15:02:44,539] {logging_mixin.py:115} INFO - [2023-07-17 15:02:44,539] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 15:02:44,539] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 15:02:44,562] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.072 seconds
[2023-07-17 16:02:14,998] {processor.py:153} INFO - Started process (PID=20857) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:02:14,999] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:02:14,999] {logging_mixin.py:115} INFO - [2023-07-17 16:02:14,999] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:02:15,016] {logging_mixin.py:115} INFO - [2023-07-17 16:02:15,015] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:02:15,016] {logging_mixin.py:115} INFO - [2023-07-17 16:02:15,016] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:02:15,016] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:02:15,043] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2023-07-17 16:02:45,785] {processor.py:153} INFO - Started process (PID=21709) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:02:45,786] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:02:45,787] {logging_mixin.py:115} INFO - [2023-07-17 16:02:45,787] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:02:45,825] {logging_mixin.py:115} INFO - [2023-07-17 16:02:45,823] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:02:45,825] {logging_mixin.py:115} INFO - [2023-07-17 16:02:45,825] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:02:45,826] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:02:45,897] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.119 seconds
[2023-07-17 16:03:16,018] {processor.py:153} INFO - Started process (PID=21939) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:03:16,019] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:03:16,020] {logging_mixin.py:115} INFO - [2023-07-17 16:03:16,020] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:03:16,053] {logging_mixin.py:115} INFO - [2023-07-17 16:03:16,052] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:03:16,054] {logging_mixin.py:115} INFO - [2023-07-17 16:03:16,054] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:03:16,054] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:03:16,107] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.095 seconds
[2023-07-17 16:03:47,222] {processor.py:153} INFO - Started process (PID=22276) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:03:47,223] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:03:47,223] {logging_mixin.py:115} INFO - [2023-07-17 16:03:47,223] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:03:47,236] {logging_mixin.py:115} INFO - [2023-07-17 16:03:47,235] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:03:47,236] {logging_mixin.py:115} INFO - [2023-07-17 16:03:47,236] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:03:47,236] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:03:47,263] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2023-07-17 16:04:17,933] {processor.py:153} INFO - Started process (PID=24327) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:04:17,934] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:04:17,935] {logging_mixin.py:115} INFO - [2023-07-17 16:04:17,935] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:04:17,949] {logging_mixin.py:115} INFO - [2023-07-17 16:04:17,949] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:04:17,950] {logging_mixin.py:115} INFO - [2023-07-17 16:04:17,950] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:04:17,950] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:04:17,976] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.047 seconds
[2023-07-17 16:04:49,760] {processor.py:153} INFO - Started process (PID=24546) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:04:49,762] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:04:49,763] {logging_mixin.py:115} INFO - [2023-07-17 16:04:49,763] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:04:49,811] {logging_mixin.py:115} INFO - [2023-07-17 16:04:49,810] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:04:49,812] {logging_mixin.py:115} INFO - [2023-07-17 16:04:49,811] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:04:49,812] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:04:49,879] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.126 seconds
[2023-07-17 16:05:20,108] {processor.py:153} INFO - Started process (PID=24797) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:05:20,109] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:05:20,110] {logging_mixin.py:115} INFO - [2023-07-17 16:05:20,110] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:05:20,144] {logging_mixin.py:115} INFO - [2023-07-17 16:05:20,143] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:05:20,144] {logging_mixin.py:115} INFO - [2023-07-17 16:05:20,144] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:05:20,144] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:05:20,195] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.094 seconds
[2023-07-17 16:05:50,437] {processor.py:153} INFO - Started process (PID=25022) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:05:50,438] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:05:50,439] {logging_mixin.py:115} INFO - [2023-07-17 16:05:50,439] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:05:50,472] {logging_mixin.py:115} INFO - [2023-07-17 16:05:50,471] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:05:50,473] {logging_mixin.py:115} INFO - [2023-07-17 16:05:50,472] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:05:50,473] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:05:50,524] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.093 seconds
[2023-07-17 16:06:21,035] {processor.py:153} INFO - Started process (PID=25565) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:06:21,037] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:06:21,038] {logging_mixin.py:115} INFO - [2023-07-17 16:06:21,037] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:06:21,075] {logging_mixin.py:115} INFO - [2023-07-17 16:06:21,074] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:06:21,075] {logging_mixin.py:115} INFO - [2023-07-17 16:06:21,075] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:06:21,076] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:06:21,133] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.106 seconds
[2023-07-17 16:06:52,029] {processor.py:153} INFO - Started process (PID=25816) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:06:52,029] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:06:52,033] {logging_mixin.py:115} INFO - [2023-07-17 16:06:52,032] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:06:52,078] {logging_mixin.py:115} INFO - [2023-07-17 16:06:52,077] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:06:52,079] {logging_mixin.py:115} INFO - [2023-07-17 16:06:52,078] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:06:52,079] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:06:52,141] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.119 seconds
[2023-07-17 16:07:22,451] {processor.py:153} INFO - Started process (PID=27049) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:07:22,452] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:07:22,454] {logging_mixin.py:115} INFO - [2023-07-17 16:07:22,453] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:07:22,493] {logging_mixin.py:115} INFO - [2023-07-17 16:07:22,492] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:07:22,494] {logging_mixin.py:115} INFO - [2023-07-17 16:07:22,494] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:07:22,494] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:07:22,561] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.117 seconds
[2023-07-17 16:07:53,528] {processor.py:153} INFO - Started process (PID=27298) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:07:53,529] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:07:53,530] {logging_mixin.py:115} INFO - [2023-07-17 16:07:53,530] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:07:53,568] {logging_mixin.py:115} INFO - [2023-07-17 16:07:53,567] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:07:53,568] {logging_mixin.py:115} INFO - [2023-07-17 16:07:53,568] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:07:53,569] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:07:53,630] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.109 seconds
[2023-07-17 16:08:24,380] {processor.py:153} INFO - Started process (PID=27523) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:08:24,381] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:08:24,382] {logging_mixin.py:115} INFO - [2023-07-17 16:08:24,382] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:08:24,421] {logging_mixin.py:115} INFO - [2023-07-17 16:08:24,420] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:08:24,421] {logging_mixin.py:115} INFO - [2023-07-17 16:08:24,421] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:08:24,422] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:08:24,490] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.117 seconds
[2023-07-17 16:08:54,863] {processor.py:153} INFO - Started process (PID=27768) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:08:54,864] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:08:54,866] {logging_mixin.py:115} INFO - [2023-07-17 16:08:54,865] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:08:54,913] {logging_mixin.py:115} INFO - [2023-07-17 16:08:54,911] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:08:54,913] {logging_mixin.py:115} INFO - [2023-07-17 16:08:54,913] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:08:54,913] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:08:54,979] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.124 seconds
[2023-07-17 16:09:25,498] {processor.py:153} INFO - Started process (PID=28313) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:09:25,499] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:09:25,499] {logging_mixin.py:115} INFO - [2023-07-17 16:09:25,499] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:09:25,514] {logging_mixin.py:115} INFO - [2023-07-17 16:09:25,513] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:09:25,514] {logging_mixin.py:115} INFO - [2023-07-17 16:09:25,514] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:09:25,514] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:09:25,534] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2023-07-17 16:09:55,961] {processor.py:153} INFO - Started process (PID=28562) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:09:55,962] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:09:55,964] {logging_mixin.py:115} INFO - [2023-07-17 16:09:55,963] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:09:56,007] {logging_mixin.py:115} INFO - [2023-07-17 16:09:56,005] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:09:56,007] {logging_mixin.py:115} INFO - [2023-07-17 16:09:56,007] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:09:56,007] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:09:56,063] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.109 seconds
[2023-07-17 16:10:26,391] {processor.py:153} INFO - Started process (PID=29028) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:10:26,393] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:10:26,394] {logging_mixin.py:115} INFO - [2023-07-17 16:10:26,394] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:10:26,432] {logging_mixin.py:115} INFO - [2023-07-17 16:10:26,431] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:10:26,432] {logging_mixin.py:115} INFO - [2023-07-17 16:10:26,432] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:10:26,432] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:10:26,493] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.110 seconds
[2023-07-17 16:10:56,575] {processor.py:153} INFO - Started process (PID=29252) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:10:56,576] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:10:56,577] {logging_mixin.py:115} INFO - [2023-07-17 16:10:56,577] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:10:56,619] {logging_mixin.py:115} INFO - [2023-07-17 16:10:56,618] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:10:56,619] {logging_mixin.py:115} INFO - [2023-07-17 16:10:56,619] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:10:56,619] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:10:56,675] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.106 seconds
[2023-07-17 16:11:27,595] {processor.py:153} INFO - Started process (PID=29482) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:11:27,596] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:11:27,596] {logging_mixin.py:115} INFO - [2023-07-17 16:11:27,596] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:11:27,609] {logging_mixin.py:115} INFO - [2023-07-17 16:11:27,609] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:11:27,610] {logging_mixin.py:115} INFO - [2023-07-17 16:11:27,610] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:11:27,610] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:11:27,630] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2023-07-17 16:11:58,090] {processor.py:153} INFO - Started process (PID=29762) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:11:58,091] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:11:58,091] {logging_mixin.py:115} INFO - [2023-07-17 16:11:58,091] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:11:58,109] {logging_mixin.py:115} INFO - [2023-07-17 16:11:58,108] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:11:58,109] {logging_mixin.py:115} INFO - [2023-07-17 16:11:58,109] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:11:58,109] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:11:58,135] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2023-07-17 16:12:29,085] {processor.py:153} INFO - Started process (PID=30147) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:12:29,086] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:12:29,086] {logging_mixin.py:115} INFO - [2023-07-17 16:12:29,086] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:12:29,104] {logging_mixin.py:115} INFO - [2023-07-17 16:12:29,104] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:12:29,104] {logging_mixin.py:115} INFO - [2023-07-17 16:12:29,104] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:12:29,105] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:12:29,131] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2023-07-17 16:12:59,391] {processor.py:153} INFO - Started process (PID=30390) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:12:59,392] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:12:59,393] {logging_mixin.py:115} INFO - [2023-07-17 16:12:59,393] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:12:59,408] {logging_mixin.py:115} INFO - [2023-07-17 16:12:59,408] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:12:59,408] {logging_mixin.py:115} INFO - [2023-07-17 16:12:59,408] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:12:59,408] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:12:59,442] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
[2023-07-17 16:13:29,991] {processor.py:153} INFO - Started process (PID=30615) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:13:29,991] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:13:29,992] {logging_mixin.py:115} INFO - [2023-07-17 16:13:29,992] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:13:30,003] {logging_mixin.py:115} INFO - [2023-07-17 16:13:30,003] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:13:30,003] {logging_mixin.py:115} INFO - [2023-07-17 16:13:30,003] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:13:30,004] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:13:30,022] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-07-17 16:14:00,736] {processor.py:153} INFO - Started process (PID=30844) to work on /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:14:00,737] {processor.py:641} INFO - Processing file /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-07-17 16:14:00,738] {logging_mixin.py:115} INFO - [2023-07-17 16:14:00,738] {dagbag.py:507} INFO - Filling up the DagBag from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:14:00,750] {logging_mixin.py:115} INFO - [2023-07-17 16:14:00,749] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-07-17 16:14:00,750] {logging_mixin.py:115} INFO - [2023-07-17 16:14:00,750] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-07-17 16:14:00,750] {processor.py:653} WARNING - No viable dags retrieved from /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-07-17 16:14:00,768] {processor.py:161} INFO - Processing /home/lucas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
